src/components/Books.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Books = () => (
+  <div>
+    <h2>Books List</h2>
+    <section className="bookSection" />
+  </div>
```

Comment:
- [ ] It will be nice to make your BookList component render a list of books using a Book component([see screenshot](https://user-images.githubusercontent.com/65476715/135070266-0d3c4c9b-f050-4fef-88f4-2597ee867a74.png)). An array of books objects could be of help.